src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -375,23 +353,23 @@ private void determineTerritoriesToAttack(final Map<Territory, ProTerritory> att
     while (true) {
       final List<ProTerritory> territoriesToTryToAttack = prioritizedTerritories.subList(0, numToAttack);
       ProLogger.debug("Current number of territories: " + numToAttack);
-      tryToAttackTerritories(attackMap, new ProMoveOptions(utils, battleUtils), unitAttackMap,
-          territoriesToTryToAttack, transportMapList, transportAttackMap, bombardMap, new ArrayList<Unit>());
+      tryToAttackTerritories(attackMap, new ProMoveOptions(), unitAttackMap, territoriesToTryToAttack,
+          transportMapList, transportAttackMap, bombardMap, new ArrayList<Unit>());
 
       // Determine if all attacks are successful
       boolean areSuccessful = true;
       for (final ProTerritory patd : territoriesToTryToAttack) {
         final Territory t = patd.getTerritory();
         if (patd.getBattleResult() == null) {
-          patd.setBattleResult(battleUtils.estimateAttackBattleResults(player, t, patd.getUnits(),
+          patd.setBattleResult(ProBattleUtils.estimateAttackBattleResults(player, t, patd.getUnits(),
               patd.getMaxEnemyDefenders(player, data), patd.getBombardTerritoryMap().keySet()));
         }
         ProLogger.trace(patd.getResultString() + " with attackers: " + patd.getUnits());
         final double estimate =
-            battleUtils.estimateStrengthDifference(t, patd.getUnits(), patd.getMaxEnemyDefenders(player, data));
+            ProBattleUtils.estimateStrengthDifference(t, patd.getUnits(), patd.getMaxEnemyDefenders(player, data));
         final ProBattleResult result = patd.getBattleResult();
         if (!patd.isStrafing() && estimate < patd.getStrengthEstimate()
-            && (result.getWinPercentage() < MIN_WIN_PERCENTAGE || !result.isHasLandUnitRemaining())) {
+            && (result.getWinPercentage() < ProData.minWinPercentage || !result.isHasLandUnitRemaining())) {
```

Comment:
This is a good opportunity to tuck some more functionality into ProData. Instead of reaching into ProData and using a data value directly: `ProData.minWinPercentage`, perhaps the boolean value of the if statement could be computed by ProData.

For example `if( ProData.likelyToWin(  patd, result)) { areSuccesful = true };`